app/uk/gov/hmrc/exports/services/notifications/receiptactions/NotificationReceiptActionsRunner.scala

```diff
@@ -60,18 +60,14 @@ class NotificationReceiptActionsRunner @Inject() (
         unparsedNotificationWorkItemRepository.complete(workItem.id, Succeeded)
       }
       .recoverWith { case throwable =>
-        val isExternalAmendmentException = throwable.isInstanceOf[ExternalAmendmentException]
-        if (isExternalAmendmentException) logger.warn(throwable.getMessage)
-
         if (workItem.failureCount + 1 >= config.parsingWorkItemsRetryLimit) {
           val msg = s"[UnparsedNotification parsing] Processing of id(${workItem.item.id}) cancelled as max retries reached!"
-          if (isExternalAmendmentException) logger.error(msg) else logger.error(msg, throwable)
+          if (throwable.isInstanceOf[ExternalAmendmentException]) logger.error(s"$msg Last error was:\n\t${throwable.getMessage}")
```

Comment:
Just thinking that if this is the 10th plus attempt that has failed then we should always `logger.error(msg, throwable)` for that unhappy scenario. Is our custom `ExternalAmendmentException` class even useful now? :thinking: